pages/_app.tsx

```diff
@@ -1,5 +1,10 @@
-import '../styles/globals.css'
-import type { AppProps } from 'next/app'
+import '../styles/globals.css';
+import type { AppProps } from 'next/app';
+import { assertConfiguration, configureAbly } from '@ably-labs/react-hooks';
+
+if (typeof window !== "undefined") { // Browser only
```

Comment:
I've consistently used `useEffect(() => {}, [])` to only execute code on the client, so I'll stick with that.

Honestly, both approaches work but I'm not sure what the recommended approach for Ably should be.